src/utils/os_zephyr.c

```diff
@@ -18,6 +18,13 @@ int clock_gettime(clockid_t clock_id, struct timespec *ts);
 #include "includes.h"
 #include "os.h"
 
+#if defined(CONFIG_NOCACHE_MEMORY)
+K_HEAP_DEFINE_NOCACHE(wifi_nm_wpa_supplicant_mem_pool, CONFIG_WIFI_NM_WPA_SUPPLICANT_HEAP);
+#else
+K_HEAP_DEFINE(wifi_nm_wpa_supplicant_mem_pool, CONFIG_WIFI_NM_WPA_SUPPLICANT_HEAP);
+#endif /* CONFIG_NOCACHE_MEMORY */
+#define WORD_SIZE 4
```

Comment:
we don't need `nocache` memory for supplicant.

Reply:
Fixed